xmtp_networking/src/grpc_api_helper.rs

```diff
@@ -74,9 +74,9 @@ impl Client {
             let connector = get_tls_connector();
 
             let tls_conn = hyper::Client::builder()
-                .pool_idle_timeout(Duration::from_secs(30))
-                .http2_keep_alive_interval(Duration::from_secs(10))
-                .http2_keep_alive_timeout(Duration::from_secs(5))
+                .pool_idle_timeout(Duration::from_secs(5))
+                .http2_keep_alive_interval(Duration::from_secs(3))
+                .http2_keep_alive_timeout(Duration::from_secs(2))
```

Comment:
I wonder if we want to keep the timeout higher here, just so that it doesn't incorrectly abort connections if you're on a high latency device (bad mobile connection). I don't think this number has any bearing on the speed we detect disconnects (pretty sure the keepalive will fail immediately if you are offline).

Reply:
Ya I wasn't really sure which one of these numbers actually did the trick from the previous PR so totally cool to keep some of these higher.

Reply:
Yeah so the interval is how often a grpc ping is sent, the timeout is the delay after which the connection is killed. More info here: https://grpc.io/docs/guides/keepalive/#common-situations-where-keepalives-can-be-useful. If the delay while testing was always 5s or less, then it seems likely that it fails immediately on the next ping, like neekolas says.